degov.yml

```diff
@@ -1,111 +1,68 @@
-name: DeGov Demo DAO
-code: degov-demo-dao
-logo: https://pbs.twimg.com/profile_images/1912415584257474560/GznnScP3_400x400.jpg
-siteUrl: https://degov-dev.vercel.app
-offChainDiscussionUrl: https://github.com/ringecosystem/degov/discussions
-description: |
-  It's a demo DAO for developers and users to experience DeGov's features, including the
-  latest innovations features like AI agents, token wrapping, and more. The DAO is built
-  on the Darwinia chain, an Ethereum-compatible chain.
-# The following fields in the links are optional and will be used to display basic information about the DAO
+name: Unlock Protocol
```

Comment:
Please do not update degov.yml, this is demo config

Reply:
That commit is made by me. It used to test the display in different DAOs in the vercel preview. Will revert when it's ready for merging.